addons/mass_mailing/static/src/scss/mass_mailing.ui.scss

```diff
@@ -113,6 +114,10 @@ html:not(.o_mass_mailing_iframe), body:not(.o_mass_mailing_iframe), html.o_fulls
             display: block;
         }
 
+        h5 {
+            font-size: 1.25rem!important;
+        }
```

Comment:
This looks taken from the media-query below.

However, in the media-query, it was just below "o_mail_theme_selector_new".
Here, it's nested under "o_mail_theme_selector_new .dropdown-item".

Is that intended?

Reply:
Thanks, no it was not intended. The idea was indeed to remove it from the media-query and make it default (to make the title bigger). I have retested, I haven’t seen any changes and anyway, it is limited to that section “o_mail_theme_selector_new”.